agent_benchmark_runner.py

```diff
@@ -763,10 +772,7 @@ def _percentile(arr: List[float], p: float) -> float:
         return arr[f] + (k - f) * (arr[c] - arr[f])
 
 
-# ------------------ Example CLI usage ------------------
-
-if __name__ == "__main__":
-    import argparse
+def main():
```

Comment:
Create a main() method at global scope (for now).

This resolves some clashes of local variables with globals